plugins/cms-export/src/assert.ts

```diff
@@ -1,3 +1,8 @@
-export function assertNever(x: never, error?: unknown): never {
-  throw error || new Error((x as unknown) ? `Unexpected value: ${x}` : "Application entered invalid state")
-}
+/**
+ * A utility function that does nothing but makes TypeScript check for the never type.
+ *
+ * For example, sometimes something that should never happen is expected to
+ * happen, like during a rollback. To prevent unwanted crashers use
+ * `shouldBeNever` instead of `assertNever`.
```

Comment:
In the context of this plugin its not about rollbacks but more about being forward compatible when union types are being extended

Reply:
Yeah, that message just got copy-pasted. I'll improve it 👌